ShadowDOM/explainer.md

```diff
@@ -25,9 +25,11 @@ content location of future work and discussions.
 * Expected venue: [Web Components CG](https://w3c.github.io/webcomponents-cg/)
 * Current version: this document
 ## Table of Contents
+- [Declarative shadow DOM style sharing](#declarative-shadow-dom-style-sharing)
```

Comment:
I don't know why this updated - I didn't edit it. There must be a Git hook or something.

Reply:
🤷‍♂️I guess there's a hook for this -- it looks correct to me.